### PR TITLE
[NUI] Support to get Marker list information from lottie

### PR DESCRIPTION
**Pull request by hinohie**
Let we allow to get the list of markers what current lottie image has now.

And also, Implement min/max frame index by marker, which is legacy feature of VD FLUX code.

Relative dali patches :
https://review.tizen.org/gerrit/c/platform/core/uifw/dali-adaptor/+/301038
https://review.tizen.org/gerrit/c/platform/core/uifw/dali-toolkit/+/301040
https://review.tizen.org/gerrit/c/platform/core/uifw/dali-extension/+/301039

### CODE REVIEW

File: src/Tizen.NUI/src/public/BaseComponents/AnimatedVectorImageView.cs [hunk 1]
```diff
@@ -581,6 +609,37 @@ public enum AnimationStates
 
 
         #region Internal
+        internal void GetFrameValueFromMarkerInfo()
+        {
+            // Note : let we insure to get marker frame value only one time per each frame marker setter
+            if (minimumFrameMarker != null)
+            {
+                var markerInfoList = GetMarkerInfo();
```
[the remark this answers is not in the document]
Comment (hinohie) [reply src/Tizen.NUI/src/public/BaseComponents/AnimatedVectorImageView.cs:617]: I think we need to consider that case. I'll update error check logic soon.

[the remark this answers is not in the document]
Comment (hinohie) [reply src/Tizen.NUI/src/public/BaseComponents/AnimatedVectorImageView.cs]: Fix done. Now we make min-max frame range as 0 ~ totalFrame-1 if we fail to found range.

[the remark this answers is not in the document]
Comment (jaehyun0cho) [reply src/Tizen.NUI/src/public/BaseComponents/AnimatedVectorImageView.cs]: Thank you very much.

**Review by jaehyun0cho — APPROVED** [review]
LGTM

**Pull request merged**